src/InputText/index.jsx

```diff
@@ -37,6 +38,7 @@ const textDefaultProps = {
   type: 'text',
   describedBy: '',
   isValid: true,
+  autoComplete: true,
```

Comment:
Need to add the `PropType.bool` for this?

Reply:
Mind adding this configuration parameter in [the README](https://github.com/edx/paragon/blob/master/src/InputText/README.md) as well? Since this is only defined in `InputText` and not in `asInput`, it should probably be explicitly noted?